README.md

```diff
@@ -201,12 +189,10 @@ Some examples of side effects:
 * printing to the screen
 * changing an original value provided to the function
 
-Obviously, functions can do more than basic arithmetic operations. They can:
+Of course, functions can do more than basic arithmetic operations. They can:
```

Comment:
The funny thing is this entire edit was supposed to remove drivel like "So" and "Of course". I failed :)